enricher/standard/src/main/resources/META-INF/fabric8/enricher-default

```diff
@@ -4,14 +4,13 @@
 # Add a default name for any resource missing
 io.fabric8.maven.enricher.standard.NameEnricher,100
 
-# Add an image from the docker build configuration if missing. Note that
-# the order of the images in the docker build section must be the same as
-# in an given external descriptor (if more than one images are used within
-# a pod)
-io.fabric8.maven.enricher.standard.ImageEnricher,101
-
 # Add a default Deployment, ReplicaSet or ReplicationController if none is given
-io.fabric8.maven.enricher.standard.DefaultControllerEnricher,102
+io.fabric8.maven.enricher.standard.DefaultControllerEnricher,101
+
+# Add image information such as name, image pull policy, environment variables
+# to a container. Controllers (like Deployment, DeploymentConfig, etc.)
+# must be already present
+io.fabric8.maven.enricher.standard.ImageEnricher,102
```

Comment:
btw, the order in this file is not important anymore (and mostly ignored, maybe we should get rid of it altogether).

The order in the enricher chain is completely defined now in the "default" profile (so that also optional enrichers can be defined in this file).